docs/src/main/sphinx/connector/jdbc-table-statistics.fragment

```diff
@@ -0,0 +1,22 @@
+### Table statistics
+
+:::{warning}
+Disabling table statistics is not recommended, as it prevents
+cost-based optimizations. Only disable for debugging purposes.
+:::
+
+The connector can use [table and column statistics](/optimizer/statistics) for
+[cost based optimizations](/optimizer/cost-based-optimizations), to improve query
+processing performance based on the actual data in the data source.
+
+The statistics are collected by the data source and retrieved by the connector.
+
+The connector may automatically update its table and column statistics. In some
+cases, you may want to force a statistics update. For example, after creating
+new columns or after changing data in the table.
+
+To force an update, run the following statement in the database:
```

Comment:
We cant say that since the syntax will be different for all of them .. unless you are saying that you can use Trino SQL ANALYZE and that will work..